checker/tests/nullness/Issue880.java

```diff
@@ -0,0 +1,22 @@
+// Test case for Issue 880:
```

Comment:
>The test case was mentioned in #880, without a reference to the jtreg test. So I added it as an explicit test.

That makes sense.  Is it just a duplicate of the jtreg test? If so, it should just be deleted.